### DISCUSSION TITLE
QTile KeyBinding Keyword for '.'

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3791] [Q&A] [answered]
Hey! i wonder what the keybinding for '.' is. I tried . and Dot but none of them work

**Comment by SweepingBishops** [comment, marked as answer]
Try running the command `xev` (if you're running xorg) in the terminal. Press the key you want in it. It'll show a lot of output, you'll have to sift through it. 
As far as I remember it was period.

**Reply by elParaguayo** [reply]
Correct. It's `period`.
